python/HeaderService/hslib_salobj.py

```diff
@@ -475,7 +475,23 @@ def collect_from_HeaderService(self):
             self.metadata['RAEND'] = ra
             self.metadata['DECEND'] = dec
         else:
-            self.log.info("No 'ELEND' and 'AZEND'")
+            self.log.info("No 'ELEND' and 'AZEND' to compute RA/DEC END")
```

Comment:
Please be specific about which cards are missing

Reply:
Fixed as requested.